relation_engine_server/utils/json_validation.py

```diff
@@ -12,12 +12,10 @@
 """
 from jsonschema import validators, Draft7Validator, FormatChecker, RefResolver
 
-from jsonschema.compat import (
```

Comment:
`jsonschema.compat` was an internal-only API and was removed in 4.X (without a mention in the release notes either...)

Reply:
Hmm I tried to do this before and ran into errors

Reply:
Okay you got the functions from somewhere else. Nice.